core/src/main/resources/hudson/model/Job/permalinks_pl.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Permalinks=Odno\u015Bniki permanentne
+Permalinks=Odno\u015Bniki
```

Comment:
This is because we don't put permanent as adjective for links and there is no need to make distinction.